<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PseudoLoginWidget</class>
 <widget class="QDialog" name="PseudoLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog, QWidget {
 background-color: rgb(101, 101, 101);
}

QProgressBar, QLineEdit {
	background: white;
	border: none;
	border-radius: 4px;
}

QProgressBar {
	border: 1px solid rgb(135, 177, 255);
}

QProgressBar, QLineEdit {
	background-color: rgb(235, 235, 235);
}

QLineEdit {
	padding: 2px;
}

QAbstractButton {
	border: 1px solid black;
	border-radius: 4px;
}

QAbstractButton:hover {
	background: #898989;
	border: 1px solid rgb(135, 177, 255);
}

QTabWidget::pane {
	border: 1px solid black;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
	border-top: 3px solid black;
	margin: 0px;
	padding: 2px;
}

QTabBar {
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
}

QTabBar::tab, QTabBar::tab:first, QTabBar::tab:middle, QTabBar::tab:last, QTabBar::tab:selected {
border: none;
 border-radius: 0px;
 background-color: transparent;
 color: black;
min-width: 8ex;
 padding: 4px;
margin-right: 3px;
}

QTabBar::tab:middle {
 border-radius: 0px;
	margin-left: 3px;
}

QTabBar::tab:last {
 border-radius: 0px;
	margin-left: 3px;
	margin-right: 1px;
}

QTabBar::tab:hover {
 border-radius: 0px;
    background-color: #898989;
}

QTabBar::tab:selected, QTabBar::tab:hover:selected {
 background-color: black;
color: white;
 border-top-left-radius: 4px;
 border-top-right-radius: 4px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Login</string>
      </attribute>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>221</width>
         <height>48</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Login [e-mail]:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Register</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Send registration request</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>183</width>
         <height>74</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Surname:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>e-mail [login]:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
